x-pack/plugins/security_solution/public/actions/toggle_column/cell_action/toggle_column.ts

```diff
@@ -66,6 +74,35 @@ export const createToggleColumnCellActionFactory = createCellActionFactory(
 
       const defaults = isTimelineScope(scopeId) ? timelineDefaults : tableDefaults;
       const { columns } = selector(store.getState(), scopeId) ?? defaults;
+      const alertTableConfigurationId = getAlertConfigIdByScopeId(scopeId);
+      if (alertTableConfigurationId) {
+        const alertsTableConfigurationRegistry =
+          services.triggersActionsUi.alertsTableConfigurationRegistry.get(
+            alertTableConfigurationId
+          );
+        alertsTableConfigurationRegistry?.actions?.toggleColumn(field.name);
```

Comment:
Is it okay to silently omit the action if `alertsTableConfigurationRegistry` or `actions` is undefined? Is it a real possibility?